internal/config/net.go

```diff
@@ -23,27 +23,30 @@ import (
 	"github.com/vdaas/vald/internal/tls"
 )
 
-// Net represent the network configuration tcp, udp, unix socket.
+// Net represents the network configuration tcp, udp, unix socket.
```

Comment:
sorry if I made a misstake 🙇 
```suggestion
// Net represents the network configuration tcp, udp, unix domain socket.
```